cmd/commands/runtime.go

```diff
@@ -380,71 +399,85 @@ func RunRuntimeInstall(ctx context.Context, opts *RuntimeInstallOptions) error {
 	}
 
 	for _, component := range rt.Spec.Components {
-		log.G(ctx).Infof("Creating component '%s'", component.Name)
-		if err = component.CreateApp(ctx, opts.KubeFactory, opts.InsCloneOpts, opts.RuntimeName, store.Get().CFComponentType, "", ""); err != nil {
-			installationErr = fmt.Errorf("failed to create '%s' application: %w", component.Name, err)
-			return installationErr
+		infoStr := fmt.Sprintf("Creating component '%s'", component.Name)
+		log.G(ctx).Infof(infoStr)
+		err = component.CreateApp(ctx, opts.KubeFactory, opts.InsCloneOpts, opts.RuntimeName, store.Get().CFComponentType, "", "")
+		appendLogToSummary(infoStr, err)
+		if err != nil {
+			return fmt.Errorf("failed to create '%s' application: %w\nfor more information: https://codefresh.io/csdp-docs/", component.Name, err)
 		}
 	}
 
 	err = installComponents(ctx, opts, rt)
+	appendLogToSummary("Installing components", err)
 	if err != nil {
-		installationErr = fmt.Errorf("failed to install components: %s", err)
-		return installationErr
+		return fmt.Errorf("failed to install components: %s\nfor more information: https://codefresh.io/csdp-docs/", err)
 	}
 
-	if err = RunGitSourceCreate(ctx, &GitSourceCreateOptions{
+	gitSrcMessage := fmt.Sprintf("Creating git source `%s`", store.Get().GitSourceName)
+	err = RunGitSourceCreate(ctx, &GitSourceCreateOptions{
 		InsCloneOpts:        opts.InsCloneOpts,
 		GsCloneOpts:         opts.GsCloneOpts,
 		GsName:              store.Get().GitSourceName,
 		RuntimeName:         opts.RuntimeName,
 		CreateDemoResources: opts.InstallDemoResources,
-	}); err != nil {
-		installationErr = fmt.Errorf("failed to create `%s`: %w", store.Get().GitSourceName, err)
-		return installationErr
+	})
+	appendLogToSummary(gitSrcMessage, err)
+	if err != nil {
+		return fmt.Errorf("failed to create `%s`: %w\nfor more information: https://codefresh.io/csdp-docs/", store.Get().GitSourceName, err)
 	}
 
 	mpCloneOpts := &git.CloneOptions{
 		Repo: store.Get().MarketplaceRepo,
 		FS:   fs.Create(memfs.New()),
 	}
 	mpCloneOpts.Parse()
-	if err = RunGitSourceCreate(ctx, &GitSourceCreateOptions{
+
+	createGitSrcMessgae := fmt.Sprintf("Creating %s", store.Get().MarketplaceGitSourceName)
+	err = RunGitSourceCreate(ctx, &GitSourceCreateOptions{
 		InsCloneOpts:        opts.InsCloneOpts,
 		GsCloneOpts:         mpCloneOpts,
 		GsName:              store.Get().MarketplaceGitSourceName,
 		RuntimeName:         opts.RuntimeName,
 		CreateDemoResources: false,
 		Exclude:             "**/images/**/*",
 		Include:             "workflows/**/*.yaml",
-	}); err != nil {
-		installationErr = fmt.Errorf("failed to create `%s`: %w", store.Get().MarketplaceGitSourceName, err)
-		return installationErr
+	})
+	appendLogToSummary(createGitSrcMessgae, err)
+	if err != nil {
+		return fmt.Errorf("failed to create `%s`: %w\nfor more information: https://codefresh.io/csdp-docs/", store.Get().MarketplaceGitSourceName, err)
```

Comment:
this 'for more information' string should really be in some utility function that decorates errors and not repeated everywhere